schema/Session.js

```diff
@@ -80,7 +80,7 @@ cube(`GroupedSession`, {
         session.retention,
         ROW_NUMBER() OVER(PARTITION BY session.user_id, session.plio_id
                           ORDER BY session.watch_time DESC) AS rank
-        FROM "public"."session"
+        FROM public.session
```

Comment:
this is not a change that made it compatible right? If not, let's revert for maintaining consistency or change others too?

Reply:
Oh this was failing in BigQuery. It was giving an error for unexpected double quotes. So I had to remove them.